src/DataWrangling/EN4/EN4_restoring.jl

```diff
@@ -36,17 +36,17 @@ Adapt.adapt_structure(to, b::EN4NetCDFBackend{N, C}) where {N, C} = EN4NetCDFBac
 
 """
     EN4NetCDFBackend(length, metadata;
-                      on_native_grid = false,
-                      cache_inpainted_data = false,
-                      inpainting = NearestNeighborInpainting(Inf))
+                     on_native_grid = false,
+                     cache_inpainted_data = false,
+                     inpainting = NearestNeighborInpainting(Inf))
 
 Represent an EN4 FieldTimeSeries backed by EN4 native netCDF files.
 Each time instance is stored in an individual file.
 """
 function EN4NetCDFBackend(length, metadata;
-                           on_native_grid = false,
-                           cache_inpainted_data = false,
-                           inpainting = NearestNeighborInpainting(Inf))
+                          on_native_grid = false,
+                          cache_inpainted_data = false,
+                          inpainting = NearestNeighborInpainting(Inf))
 
     return EN4NetCDFBackend{on_native_grid, cache_inpainted_data}(1, length, inpainting, metadata)
```

Comment:
if the backend stores the metadata, is it possible to have one backend for both ECCO and EN4? This is a minor savings in code so not crucial --- just curious

Reply:
what backend you referring to? this is part of a file I removed

Reply:
You are referring to `EN4NetCDFBackend`, right? This is not part of this PR so the comment is relevant of the Restoring refactor, right?

Reply:
oh sorry